src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -4235,4 +4235,7 @@
   <data name="OutOfMemory_StringTooLong" xml:space="preserve">
     <value>String length exceeded supported range.</value>
   </data>
-</root>
+  <data name="Arg_MustBeArrayType" xml:space="preserve">
+    <value>Type must be a array type.</value>
```

Comment:
We have `Argument_HasToBeArrayClass` that is used in similar situations - I think we can just reuse that string.